video/buffered_frame_decryptor.cc

```diff
@@ -85,7 +89,8 @@ BufferedFrameDecryptor::FrameDecision BufferedFrameDecryptor::DecryptFrame(
   // Attempt to decrypt the video frame.
   const FrameDecryptorInterface::Result decrypt_result =
       frame_decryptor_->Decrypt(cricket::MEDIA_TYPE_VIDEO, /*csrcs=*/{},
-                                additional_data, *frame,
+                                additional_data,
+                                encrypted_bitstream,
```

Comment:
The *frame passed in there was way too subtle.  What was happening was the frame (because it has a .data() and .size()) was being coerced into an rtc::ArrayView.  If I have to debug this again, I'd rather not try and figure out that magic again.  Better to just make it readable while we're in here.